src/main/java/core/mate/academy/producer/implementations/BulldozerProducer.java

```diff
@@ -0,0 +1,18 @@
+package core.mate.academy.producer.implementations;
+
+import core.mate.academy.model.Bulldozer;
+import core.mate.academy.model.Machine;
+import core.mate.academy.service.MachineProducer;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Machine> {
+    @Override
+    public List<Machine> get() {
```

Comment:
```suggestion
public class BulldozerProducer implements MachineProducer<Bulldozer> {
    @Override
    public List<Bulldozer> get() {
```
this class should be able to work only with Bulldozer